mailpile/plugins/setup_magic.py

```diff
@@ -283,6 +283,9 @@ def basic_app_config(self, session,
         if not vcard_importers.libravatar:
             vcard_importers.libravatar.append({'active': True})
             session.ui.notify(_('Enabling Libravatar image importer'))
+        if not vcard_importers.face:
+            vcard_importers.face.append({'active': True})
+            session.ui.notify(_('Enabling Face image importer'))
```

Comment:
Until this feature has been tested a bit, I am not sure it should be enabled by default.  Especially see my concerns below about security... :-)